imapclient/imapclient.py

```diff
@@ -535,7 +535,7 @@ def idle(self):
         self._idle_tag = self._imap._command('IDLE')
         resp = self._imap._get_response()
         if resp is not None:
-            raise self.Error('Unexpected IDLE response: %s' % resp)
+            raise exceptions.IDLEResponseError('Unexpected IDLE response: %s' % resp)
```

Comment:
I'm not sure about this one. It seems that it's just a command that failed. Is it different from any other command failing for a random reason?